### ISSUE TITLE
Teleport with Animals?

### ISSUE THREAD

**Issue by freakyy85** [andune/HomeSpawnPlus#93]
Can you make it so players can teleport together with their animals?

Thanks!

**Comment by sorifiend** [comment]
It is a really neat idea, if I get time I will see about adding the feature, but its very very low on my list of things to do, if anyone else wants to give it a shot then please do.

**Comment by freakyy85** [comment]
yes please - all my players want their horses to be teleported with them.
